pkgs/applications/video/gnomecast/default.nix

```diff
@@ -17,7 +17,10 @@ buildPythonApplication rec {
   ];
 
   preFixup = ''
-    gappsWrapperArgs+=(--prefix PATH : ${lib.makeBinPath [ ffmpeg_3 ]})
+    gappsWrapperArgs+=( \
+      --set GDK_PIXBUF_MODULE_FILE ${librsvg}/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache \
```

Comment:
The `wrapGAppsHook` should do this automatically. The reason this does not work here is that `https://github.com/NixOS/nixpkgs/pull/79987` sets `strictDeps = true` which does not work very well with setup hooks. I would suggest using `strictDeps = false`: https://nixos.org/manual/nixpkgs/stable/#ssec-gnome-hooks-gobject-introspection